src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -59,6 +63,35 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AvailableSubsystems)
   }
 }
 
+
+TEST_F(Cgroups2Test, ROOT_CGROUPS2_AssignProcessToCgroup)
+{
+  string cgroup = "test";
+  ASSERT_SOME(cgroups2::create(cgroup));
```

Comment:
We need to add cleanup logic into the Cgroups2Test fixture, otherwise if this test leaves early nothing cleans up the test cgroup, can you follow up on this in a separate PR?